### ISSUE TITLE
the training set of multithumos?

### ISSUE THREAD

**Issue by Wyz2927** [dairui01/MS-TCT#2]
hello, how about the training set of multithumos? Is it the same as charades? I have trained my own model using the following training set, but only get 40.2136 f-map. Do you konw what is the problem?
![image](https://user-images.githubusercontent.com/96815275/176811730-44e3a105-adf9-433d-aa6d-a18986e41d18.png)
![image](https://user-images.githubusercontent.com/96815275/176811829-d963263e-7b49-4c7b-8b04-c405c534978e.png)

![image](https://user-images.githubusercontent.com/96815275/176811765-d7378365-b014-41f8-ab6c-a5784711a534.png)

**Comment by dairui01** [comment]
Thanks for the interest in the MS-TCT. 
From the provided parameters, it may need to change the batch size and initial learning rate. 
You can try batch_size = 8 and lr = 0.0005. As MultiTHUMOS has a difference in terms of video length compared to Charades, so you may need to change the number of batch sizes. Also, the I3D features that are not pre-trained on the MultiTHUMOS training set may also hamper the performance.

**Comment by Wyz2927** [comment]
> Thanks for the interest in the MS-TCT. From the provided parameters, it may need to change the batch size and initial learning rate. You can try batch_size = 8 and lr = 0.0005. As MultiTHUMOS has a difference in terms of video length compared to Charades, so you may need to change the number of batch sizes. Also, the I3D features that are not pre-trained on the MultiTHUMOS training set may also hamper the performance.

Thank you so much for your quick reply. I have trained my model with your suggested setting and I got 43.8647% full-val-fmap at the 53rd epoch, even higher than you reported in the paper. 
![image](https://user-images.githubusercontent.com/96815275/176853846-9ab2b798-3dc3-4cfe-9bd5-e23da847e6b2.png)
And I have another question. I found that you also report the map score of MLAD on multithumos in your paper. I have tried to reproduce this results but only get around 37.5 map. Could you please share the experimental setup if it is convenient? 
I use the following settings.
![image](https://user-images.githubusercontent.com/96815275/176855489-4d4aebd3-2fa3-4f3e-bcdb-c2d673b98aed.png)

**Comment by dairui01** [comment]
Thanks for testing the parameters. As I have refined a version of the code after the CVPR submission, thus it may perform better than the one reported in the paper. 

Regarding the MLAD on MultiTHUMOS, I am also unable to reproduce the result reported in that paper. So in the table of "comparison with SOTA", I just reuse their paper's number directly. Maybe you can raise this issue in MLAD's repo.

**Comment by wzy246532** [comment]
Dear Wyz2927,

I have seen the results of your training on the MultiTHUMOS dataset using the MS-TCT model. I would appreciate it if you could share the feature data extracted by MultiTHUMOS on I3D for MS-TCT input.

Best wishes!

**Comment by Wyz2927** [comment]
> Dear Wyz2927,
> 
> I have seen the results of your training on the MultiTHUMOS dataset using the MS-TCT model. I would appreciate it if you could share the feature data extracted by MultiTHUMOS on I3D for MS-TCT input.
> 
> Best wishes!

Another paper MLAD has provided the features for MultiTHUMOS. you can find it in https://github.com/ptirupat/MLAD.

**Comment by wzy246532** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dairui01/MS-TCT"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年12月1日(星期四) 下午5:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dairui01/MS-TCT] the training set of multithumos? (Issue #2)





  
Dear Wyz2927,
 
I have seen the results of your training on the MultiTHUMOS dataset using the MS-TCT model. I would appreciate it if you could share the feature data extracted by MultiTHUMOS on I3D for MS-TCT input.
 
Best wishes!
  
Another paper MLAD has provided the features for MultiTHUMOS. you can find it in https://github.com/ptirupat/MLAD.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.

Thank you very much for your quick reply. I have downloaded the corresponding rgb features, which are rgb_test.h5 and rgb_val.h5 respectively. But I ran into a new problem, they are different from charades' features, I don't know how to deal with them? Are they treated in the same way as charades features?

Best wishes!

**Comment by shiyi-z** [comment]
hi the link of the features for MultiTHUMOS provided by MLAD was invalid, I wonder that do you have a new link or can you share the profile of the data feature ,i would appreciate it !

**Comment by 1220894232** [comment]
Hi there,
I wanted to ask you about the MultiTHUMOS dataset and its features that you've shared on your GitHub repository at https://github.com/ptirupat/MLAD. I've been trying to access the features through the provided link, but it appears to be broken. I was wondering if there is a new link available or another way to obtain these features?
I appreciate any help you can provide. Thank you in advance.
Best regards
